engines/sci/engine/script_patches.cpp

```diff
@@ -1988,9 +1999,9 @@ static const uint16 gk1GranChairFlickerPatch[] = {
 	0x38, PATCH_UINT16(0x00af),         // pushi 00af
 	0x39, 0x75,                         // pushi 75
 	0x81, 0x00,                         // lag 0
+	0xa3, 0x00,                         // sal local0 [ local0 just needs to be non-zero ]
```

Comment:
Why does it need to be non-zero? Please add a comment here

Reply:
Good call, I should have also explained the setCel/cel seelctor change so I updated the comments for both in the signature and patch.